tests/e2e/scale/test_pv_scale_and_respin_ceph_pods.py

```diff
@@ -183,5 +183,5 @@ def test_pv_scale_out_create_pvcs_and_respin_ceph_pods(
 
         # Added sleep for test case run time and for capturing memory leak after scale
         time.sleep(test_run_time)
-        # TODO: Skipping memory leak analysis because of bz 1750328
-        # helpers.memory_leak_analysis(median_dict)
+        assert utils.ceph_health_check(delay=180), "Failed, Ceph health NOT_OK after ceph pod respin"
```

Comment:
no need assert as this function returns nothing but raise exception in case of failure

Reply:
Updated the TC